railties/lib/rails/generators/rails/app/templates/config/locales/en.yml

```diff
@@ -4,30 +4,31 @@
 #
 # To use the locales, use `I18n.t`:
 #
-#     I18n.t "hello"
```

Comment:
Can you change back all the quotes from single to double? Rails defaults to always use double quotes.

Reply:
Sure, but do you mind if I also update the comment to just say `quotes` (or `double quotes`)? It seemed a bit funny to say `surround them with single quotes` and then include an example that used double-quotes.